templates/jenkins.yaml.erb

```diff
@@ -3,10 +3,6 @@ tool:
     installations:
     - home: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\MSBuild\\15.0\\Bin"
       name: "msbuild-vs2017buildtools-15"
-plugins:
```

Comment:
Why have we removed this?

Reply:
It's already predefined settings in Jenkins.